src/utils/stdutils.ts

```diff
@@ -0,0 +1,33 @@
+import { type StandardLibraryType } from 'services/CommonService'
+
+const getCapabilityTitle = (
+  values: number[],
+  StandardJson: StandardLibraryType
```

Comment:
naming conventions: variables, constants, arguments and props start with lower case - types and interfaces with UpperCase